bin/run-vitest-all.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Get all workspace names
+workspaces=$(yarn workspaces --json info | jq -r 'if .type == "log" then .data | fromjson | keys[] else empty end')
```

Comment:
I had to install "jq" on my mac .. 

Could we do without ? Is there an alternative ?

Reply:
Not as reliable, but it would work:

```bash
workspaces=$(yarn workspaces info | grep -o '@symfony/[^"]*')
```

Reply:
If it works and does not take 5seconds more.. i think it's better not to require tools on a DX pov. WDYT ?

Reply:
Yup! If it works, then it's reliable enough, as we control (via our packages) the output of `yarn workspaces info`. So it shouldn't suddenly stop working, and it's just a dev/testing tool. So let's change it